packages/xo-web/src/common/intl/messages.js

```diff
@@ -1154,12 +1155,14 @@ const messages = {
   vmCpuLimitsLabel: 'CPU limits',
   vmCpuTopology: 'Topology',
   vmChooseCoresPerSocket: 'Default behavior',
-  vmCoresPerSocket:
+  vmSocketsWithCoresPerSocket:
     '{nSockets, number} socket{nSockets, plural, one {} other {s}} with {nCores, number} core{nCores, plural, one {} other {s}} per socket',
   vmCoresPerSocketNone: 'None',
-  vmCoresPerSocketIncorrectValue: 'Incorrect cores per socket value',
-  vmCoresPerSocketIncorrectValueSolution:
-    'Please change the selected value to fix it.',
+  vmCoresPerSocket:
+    '{nCores, number} core{nCores, plural, one {} other {s}} per socket',
+  vmCoresPerSocketNotDivisor: "Not a divisor of the VM's max CPUs",
+  vmCoresPerSocketExceedsLimit:
+    'The selected value exceeds limits ({maxSockets, number} sockets {maxCores, number} cores)',
```

Comment:
exceeds *the* limits (n sockets *and* m cores)

Reply:
What's your idea behind using `n` instead of the named variable `maxSockets`?